src/posts/tracker/basics.ts

```diff
@@ -138,7 +140,7 @@ The good news is that modeling a custom person is almost exactly the same as mod
 1. **Ask them about their behavior in the last 10 days**
     *   The more you know about what they have done in the last 10 days, the more accurate you can be. Since we are overestimating when there is uncertainty, quite often being more precise will reduce the person’s risk score.
     *   **Tip:** If you wish, you can send them our [recent activity questions](/questions) as a way to make the process clearer and smoother.
-    *   **Note:** Modeling someone's activities for the last 10 days is 
+    *   **Note:** Modeling someone's activities for the last 10 days is only accurate when their risk score is relatively low. If they have done any higher risk activities (around 100x your weekly budget) in the lsat 21 days, you may want to include those in your modeling. You can read more about doing so in the [FAQ](./faq#whats-up-with-the-0-7-day--range-and-2-9-day-range-what-are-they-for-why-the-difference).
```

Comment:
typo lsat -> last.

And I'd say 10 days is accurate if they do the same thing every week; Asking if they've done anything out of the ordinary in the last 21 days might be better / easier to understand

Reply:
Fixed. Thanks! 🙏